### ISSUE TITLE
[JENKINS-33822] New Item creation keyboard navigation does not work

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-15#5603]
<p>We'll need to make sure keyboard navigation works on this. Currently the radio buttons are display: none, which makes the browser not allow selection.</p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/kzantow.png" title="kzantow's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kzantow">kzantow</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-33822" target="_blank">New Item creation keyboard navigation does not work</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/recena.png" title="recena's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=recena">recena</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Major
<li><b>component(s)</b>: core
<li><b>label(s)</b>: 2.0, 2.0-beta, community-bee, regression
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2016-05-03T13:37:55+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 5
<li><b>imported</b>: 2025-11-16
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>We'll need to make sure keyboard navigation works on this. Currently the radio buttons are display: none, which makes the browser not allow selection.</p></pre>
</details>
</details>
<ul><li><i>environment</i>: <code>2.0-beta</code></li></ul>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>swashbuck1r's <a href="https://issues.jenkins.io/browse/JENKINS-33822?focusedId=252861&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-252861">comment</a>:</i></summary>


<details><summary><sub><i>Raw content of original comment:</i></sub></summary>

<pre>
<p>This is a regression in keyboard nav capabilities for new item types.</p>

<p>Note:  the Jenkins 1.x version had it's own flaws...since tabbing would select the copy-item textbox, so we shouldn't completely mimic the 1.x behavior.</p>
</pre>
</details>

</details>
<p>This is a regression in keyboard nav capabilities for new item types.</p>

<p>Note:  the Jenkins 1.x version had it's own flaws...since tabbing would select the copy-item textbox, so we shouldn't completely mimic the 1.x behavior.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/recena.png" title="recena's avatar" /> recena's <a href="https://issues.jenkins.io/browse/JENKINS-33822?focusedId=253805&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-253805">comment</a>:</i></summary>


<details><summary><sub><i>Raw content of original comment:</i></sub></summary>

<pre>
<p>This is not a regression since never worked fine.</p>
</pre>
</details>

</details>
<p>This is not a regression since never worked fine.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/recena.png" title="recena's avatar" /> recena's <a href="https://issues.jenkins.io/browse/JENKINS-33822?focusedId=254009&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-254009">comment</a>:</i></summary>


<details><summary><sub><i>Raw content of original comment:</i></sub></summary>

<pre>
<p>There is a PR verified and reviewed <a href="https://github.com/jenkinsci/jenkins/pull/2250" class="external-link" target="_blank" rel="nofollow noopener">here</a>.</p>
</pre>
</details>

</details>
<p>There is a PR verified and reviewed <a href="https://github.com/jenkinsci/jenkins/pull/2250" class="external-link" target="_blank" rel="nofollow noopener">here</a>.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/scm_issue_link.png" title="scm_issue_link's avatar" /> scm_issue_link's <a href="https://issues.jenkins.io/browse/JENKINS-33822?focusedId=256233&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-256233">comment</a>:</i></summary>


<details><summary><sub><i>Raw content of original comment:</i></sub></summary>

<pre>
<p>Code changed in jenkins
User: Manuel Recena
Path:
 core/src/main/resources/hudson/model/View/newJob.jelly
 war/src/main/js/add-item.js
 war/src/main/js/widgets/add/addform.less
<a href="http://jenkins-ci.org/commit/jenkins/ece6606066abcf33dbff7be42b35e78253cad267" class="external-link" target="_blank" rel="nofollow noopener">http://jenkins-ci.org/commit/jenkins/ece6606066abcf33dbff7be42b35e78253cad267</a>
Log:
  <a href="https://issues.jenkins.io/browse/JENKINS-33822" title="New Item creation keyboard navigation does not work" class="issue-link" data-issue-key="JENKINS-33822"><del>JENKINS-33822</del></a> Create a new item using the keyboard</p>


</pre>
</details>

</details>
<p>Code changed in jenkins<br/>
User: Manuel Recena<br/>
Path:<br/>
 core/src/main/resources/hudson/model/View/newJob.jelly<br/>
 war/src/main/js/add-item.js<br/>
 war/src/main/js/widgets/add/addform.less<br/>
<a href="http://jenkins-ci.org/commit/jenkins/ece6606066abcf33dbff7be42b35e78253cad267" class="external-link" target="_blank" rel="nofollow noopener">http://jenkins-ci.org/commit/jenkins/ece6606066abcf33dbff7be42b35e78253cad267</a><br/>
Log:<br/>
  <a href="https://issues.jenkins.io/browse/JENKINS-33822" title="New Item creation keyboard navigation does not work" class="issue-link" data-issue-key="JENKINS-33822"><del>JENKINS-33822</del></a> Create a new item using the keyboard</p>
